src/consts.h

```diff
@@ -14,7 +14,10 @@
 // General Constants
 //------------------
 
-#define   VERSION            51011                                             //(5.1.011)
+// Update VERSION and SEMVERSION Simultaneously 
+#define   VERSION            51013          // Eventually will be deprecated. 
+#define   SEMVERSION         "5.1.13.dev0"  // Semantic Version
```

Comment:
Actually I think it is more like 

`5.1.13dev0`

We could store this as struct with major, minor, patch ?

```
struct VERSION_INFO {
   char  major[3];
   char  minor[3];
   char  patch[10];
};
```

Or something?